src/App/Application.cpp

```diff
@@ -435,9 +436,11 @@ void Application::renameDocument(const char *OldName, const char *NewName)
 
 Document* Application::newDocument(const char * Name, const char * UserName, bool createView, bool tempDoc)
 {
+    QString L10nName = QObject::tr("Unnamed");
     // get a valid name anyway!
     if (!Name || Name[0] == '\0')
-        Name = "Unnamed";
+        Name = qUtf8Printable(L10nName);

```

Comment:
You cannot store the pointer created by `qUtf8Printable`, it falls out of scope immediately following the statement.